h/services/annotation_json.py

```diff
@@ -99,9 +99,17 @@ def present(self, annotation: Annotation):
         if annotation.references:
             model["references"] = annotation.references
 
+        if with_metadata and annotation.slim.meta:
```

Comment:
The LMS side needs the metadata we have in the DB to know for which assignment this is.

Making it configurable so we only expose it on the LMS queue.

Reply:
How are we going to decide if `with_metadata` is True or False?

Do you see any potential issue on publishing the metadate unconditionally so that there are less logic branches to take care of?

Reply:
With the current design that would expose the metadata in all public API responses. I reckon is best to keep it just for this queue.

Not that would expose any secret data or similar but I rather keep the API response untouched.

For now we only use `with_metadata` to publish on the queue.

Reply:
Oh, I thought this logic was specific for serializing  for the queue publishing. If this code is used to expose the annotation in the API, then I agree it's better to not change it.

My question was if it made sense to always publish the annotation in the queue with metadata, which seems to be in fact the case, so all good.